CHANGES.md

```diff
@@ -33,6 +33,27 @@ To be released.
         by `BlockChain<T>`.
  -  Unused parameter `currentTime` removed from `BlockChain<T>.Append()`
     [[#1462], [#1465]]
+ -  `BlockHeader.Timestamp` property's type became `DateTimeOffset`
+    (was `string`).  [[#1470]]
+ -  `BlockHeader.Nonce` property's type became `Nonce` (was
+    `ImmutableArray<byte>`).  [[#1470]]
+ -  `BlockHeader.Miner` property's type became `Address` (was
+    `ImmutableArray<byte>`).  [[#1470]]
+ -  `BlockHeader.PreviousHash` property's type became `BlockHash?` (was
+    `ImmutableArray<byte>`).  [[#1470]]
+ -  `BlockHeader.TxHash` property's type became `HashDigest<SHA256>?` (was
+    `ImmutableArray<byte>`).  [[#1470]]
+ -  `BlockHeader.Hash` property's type became `BlockHash` (was
+    `ImmutableArray<byte>`).  [[#1470]]
+ -  `BlockHeader.StateRootHash` property's type became `HashDigest<SHA256>?`
+    (was `ImmutableArray<byte>`).  [[#1470]]
+ -  Removed `BlockHeader(int, long, string, ImmutableArray<byte>,
+    ImmutableArray<byte>, long, BigInteger, ImmutableArray<byte>,
+    ImmutableArray<byte>, ImmutableArray<byte>, ImmutableArray<byte>,
+    ImmutableArray<byte>)` constructor.  Use `BlockHeader(int, long,
+    DateTimeOffset, Nonce, Address, long, BigInteger, BlockHash?,
+    HashDigest<SHA256>?, BlockHash, ImmutableArray<byte>, HashDigest<SHA256>?)`
+    constructor instead.  [[#1470]]
```

Comment:
I think grouping with an additional level of itemization might make this more readable.

Reply:
Amended!